project.mml

```diff
@@ -2278,6 +2277,30 @@ Layer:
     properties:
       cache-features: true
       minzoom: 14
+# Render railway crossings except if they concern tramways only
+  - id: railway-crossings
+    geometry: point
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: &railway-crossing_sql |-
+        (SELECT DISTINCT ON (crossing.way)
+            'railway_' || crossing.railway AS feature,
+            crossing.way AS way,
+            crossing.access AS access,
+            crossing.railway AS type,
+            crossing.railway AS railway,
+            track.railway AS int_lc_type
+          FROM planet_osm_point crossing
+            JOIN planet_osm_line track
+              ON ST_DWithin(crossing.way, track.way, 0.1) -- Assumes Mercator
+          WHERE crossing.railway IN ('crossing', 'level_crossing')
+            AND track.railway IS NOT NULL AND track.railway <> 'tram'
```

Comment:
That means all railway crossings that are wrongly mapped not on any kind of railway (tram or otherwise) are hidden and not visible to the map user.  That is incompatible with our goal of providing constructive mapper feedback i think.  We - as a general rule - do not hide features from rendering based on spatial analytics telling us that they are mapped incorrectly.  If a mapper maps a railway crossing in the middle of nowhere we show that because the mapper either has a good reason for that or it is an error and should be corrected and showing it helps mappers in seeing such errors.